js/Ass.js

```diff
@@ -0,0 +1,35 @@
+$(document).ready (function(){
+
+    var female1 = prompt("Please enter a female's name (1 of 2)");
+  $('#female1-1').text(female1);
+  $('#female1-2').text(female1);
+  $('#female1-3').text(female1);
```

Comment:
I like how you have prompted for a value and then used jQuery to assign. Well done!